kaizen/autofix/main.py

```
@@ -1510,11 +1167,13 @@ class AutoFix:
     LOG_LEVEL_ERROR = "error"
     ERROR_MSG_FIX_FAILED = "Failed to fix code"
     
-    def __init__(self, config: Union[Dict, 'TestConfiguration'], runner_config: Dict[str, Any]):
+    def __init__(self, config: Union[Dict, 'TestConfiguration'], runner_config: Dict[str, Any], memory=ExecutionMemory()):
```

Ruff (0.12.2)

Do not perform function call `ExecutionMemory` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)